packages/esm-appointments-app/src/appointments/scheduled/appointments-by-status.component.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
It might be nice if this could take in an array of statuses.  I could see there being a use case to combine, say, everything that is either "CheckedIn" or "Completed" in in a tab named "Honored".

Reply:
Yep, I suspect we are going to need that, but as the endpoint doesn't yet support it, I didn't add it yet.  If we need it, we can figure out whether to modify the endpoint or handle this on the client-side.